src/components/Reports/ReportEdit.tsx

```diff
@@ -141,8 +144,33 @@ export default function ReportEdit({ organization }: ReportEditProps): JSX.Eleme
     }
   };
 
+  const hasEmptyRequieredFields = (iReport: Report) => {
+    const emptySeedbankFields = iReport.seedBanks?.some((sb) => {
+      return !sb.buildStartedDate || !sb.buildCompletedDate || !sb.operationStartedDate;
+    });
+    const emptyNurseryFields = iReport.nurseries?.some((nursery) => {
+      return (
+        !nursery.buildStartedDate || !nursery.buildCompletedDate || !nursery.operationStartedDate || !nursery.capacity
+      );
+    });
+    const emptyPlantingSitesFields = iReport.plantingSites?.some((plantingSite) => {
+      return (
+        !plantingSite.totalPlantingSiteArea ||
+        !plantingSite.totalPlantedArea ||
+        !plantingSite.totalTreesPlanted ||
+        !plantingSite.totalPlantsPlanted ||
+        !plantingSite.mortalityRate
+      );
+    });
+    return emptySeedbankFields || emptyNurseryFields || emptyPlantingSitesFields;
+  };
   const submitReport = async () => {
     if (report) {
+      if (hasEmptyRequieredFields(report)) {
+        setValidateFields(true);
+        setShowAnnual(false);
```

Comment:
Maybe use the `handleBack` function here if `showAnnual` is true, so that any annual details that were entered are saved